.github/workflows/nix-action-default.yml

```diff
@@ -0,0 +1,127 @@
+jobs:
+  coq:
+    needs: []
```

Comment:
Remove

Reply:
Ah ok, it made my linter mad since empty lists aren't expected by the yaml schema.